mrp_bom_location/report/bom_structure.py

```diff
@@ -0,0 +1,89 @@
+# Copyright 2017-20 ForgeFlow S.L. (https://www.forgeflow.com)
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl).
+
+from odoo import api, models
+
+
+class BomStructureReport(models.AbstractModel):
+    _inherit = "report.mrp.report_bom_structure"
+
+    @api.model
+    def _get_bom_data(
+        self,
+        bom,
+        warehouse,
+        product=False,
+        line_qty=False,
+        bom_line=False,
+        level=0,
+        parent_bom=False,
+        index=0,
+        product_info=False,
+        ignore_stock=False,
+    ):
+        res = super()._get_bom_data(
+            bom,
+            warehouse,
+            product=product,
+            line_qty=line_qty,
+            bom_line=bom_line,
+            level=level,
+            parent_bom=parent_bom,
+            index=index,
+            product_info=product_info,
+            ignore_stock=ignore_stock,
+        )
+        line_ids = self.env["mrp.bom.line"].search([("bom_id", "=", bom.id)])
```

Comment:
```suggestion
        line_ids = bom.bom_line_ids
```

I don't undertand this search. Why don't we just read the field?